spec/immutable-data-structures.html

```diff
@@ -83,6 +83,10 @@ <h1>Record.prototype</h1>
         <p>This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: false }.</p>
       </emu-clause>
     </emu-clause>
+    <emu-clause id="sec-properties-of-record-instances">
+      <h1>Properties of Record Instances</h1>
+      <p>Record instances are ordinary objects with a *null* prototype. Symbol instances have a [[RecordData]] internal slot. The [[RecordData]] internal slot is the Record value represented by this Record object.</p>
+    </emu-clause>
```

Comment:
```suggestion
    <emu-clause id="sec-properties-of-record-objects">
      <h1>Properties of Record Objects</h1>
      <p>Record objects are ordinary objects with a *null* prototype. Record objects have a [[RecordData]] internal slot. The [[RecordData]] internal slot is the Record value represented by this Record object.</p>
    </emu-clause>
```

changing wording to use "record objects" for now.